### DISCUSSION TITLE
Setting up Radicale on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by QasimK** [QasimK/qasimk.github.io#28] [Blog Posts]
Comment on [Setting up Radicale on a Raspberry Pi](http://qasimk.io/2017/radicale-server/).

**Comment by wknds** [comment]
To use bcrypt with a raspbian-packaged radicale, you need the following packages:

    sudo apt-get install build-essential libssl-dev libffi-dev python-dev
    sudo python -m pip install –upgrade passlib
    sudo python -m pip install –upgrade bcrypt

Worked for me :-)

**Comment by QasimK** [comment]
Hi @yaw89, thanks for your suggestion. I will give installing the dependencies a go and update the article if it works out
